app/controllers/families_controller.rb

```diff
@@ -143,6 +143,22 @@ def schema
     render xml: Family.columns.map { |c| {name: c.name, type: c.type} }
   end
 
+  def directory_map
+  end
+
+  def map_view_details
+    mapable_families = Family.all.select{|f| f.mapable?}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:mapable?` as an argument to `select` instead of a block.